adafruit_wiznet5k/adafruit_wiznet5k_dhcp.py

```diff
@@ -127,6 +127,10 @@ def send_dhcp_message(self, state, time_elapsed):
         :param float time_elapsed: Number of seconds elapsed since renewal.
 
         """
+        # before making send packet, shoule init _BUFF. if not, DHCP sometimes fails, wrong padding, garbage bytes, ...
+        for i in range(len(_BUFF)):
+            _BUFF[i] = 0
```

Comment:
Could you change this to a one-liner:

`_BUFF = [0] * len(_BUFF)`

Reply:
looks very good! Thanks for your comment~

Reply:
`_BUFF[:] = b'\x00' * len(_BUFF)`
it solved